cdap-hbase-compat-base/src/main/java/co/cask/cdap/replication/StatusUtils.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.replication;
+
+import co.cask.cdap.common.conf.Constants;
+import org.apache.hadoop.conf.Configuration;
+
+import java.io.IOException;
+
+/**
+ * Utility Functions for Replication Status Coprocessors
+ */
+public final class StatusUtils {
```

Comment:
for utility classes, we usually have a private no-argument constructor to prevent people from instantiating one.

```
private StatusUtils() {
}
```